pom.xml

```diff
@@ -3,17 +3,17 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>io.gitlab.arturbosch</groupId>
-    <artifactId>sonar-kotlin</artifactId>
-    <version>1.5.0</version>
+    <groupId>io.github.detekt</groupId>
+    <artifactId>sonar-detekt</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
```

Comment:
Why is *snapshot* used here?

Reply:
I will rename it to RC1 for a first release to the update store.